app/presenters/register_api/single_application_presenter.rb

```diff
@@ -34,15 +34,11 @@ def as_json
             english_language_qualifications: application_form.english_language_qualification_details,
             other_languages: application_form.other_language_details,
             disability_disclosure: application_form.disability_disclosure,
-            gender: equality_and_diversity_data['sex'],
-            disabilities: equality_and_diversity_data['disabilities'].presence || [],
-            ethnic_group: equality_and_diversity_data['ethnic_group'],
-            ethnic_background: equality_and_diversity_data['ethnic_background'],
           },
           contact_details: contact_details,
           course: course_info_for(application_choice.offered_option),
           qualifications: qualifications,
-          hesa_itt_data: hesa_itt_data.presence || {},
```

Comment:
do we not want to default to a hash if this is empty?

Reply:
I think it's better to return an explicit `null` than a hash which could have misleading semantics like `hash[:gender] == nil`, which could imply eg the user declined to give this specific piece of information